### DISCUSSION TITLE
miktex-texworks

### DISCUSSION THREAD

**Discussion by FelixKlein2022** [MiKTeX/miktex#1096] [General]
I have installed miktex on my MacAir and it works fine and I am happy with the included Texworks. I would like to associate the .tex files with miktex-texworks,   but the usual way (right-click,  Get info, ...)   does not work since I  cannot find  miktex-texworks  on the list  of available applications.  Do you have any suggestions what to do?  Thanks

**Comment by vitorlppf** [comment]
I am having the same issue. Earlier today I installed MiKTeX on Ubuntu 20.04 and I cannot set it as the default application to edit .tex files with.
